src/modality-specific-files/genetic-descriptor.md

```diff
@@ -33,14 +33,7 @@ The definitions of these fields can be found in
 and a guide for using macros can be found at
  https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
 -->
-{{ MACROS___make_subobject_table(
-   ("objects", "metadata", "Genetics"),
-   {
-      "Dataset": "REQUIRED",
-      "Database": "OPTIONAL",
-      "Descriptors": "OPTIONAL",
-   },
-) }}
+{{ MACROS___make_subobject_table("metadata.Genetics") }}
```

Comment:
This doesn't introduce any problems, but I did notice that this table and the above text don't match. The text says that there is a `.` notation, which we have dropped, so it's actually not clear what the key to the subobject should be (i.e., `Genetics`).

Reply:
Resolved by d90d36d85647d81f2c88ca19deba61830afa50ec.